tools/icu/icu-generic.gyp

```diff
@@ -161,6 +161,46 @@
               '../../deps/icu/source/i18n/uspoof_impl.h',
               '../../deps/icu/source/i18n/uspoof_wsconf.cpp',
               '../../deps/icu/source/i18n/uspoof_wsconf.h',
+            ]}],         
+            [ 'icu_ver_major == 57', { 'sources!': [
+              ## Strip out the following for ICU 55 only.
```

Comment:
Yes it should say 57. 

I'm wondering if I should just fold this into #6088. Basically just add the two `configure` lines fixing the URL from this PR into #6088 and close this one. 

Thoughts?